shakenfist/baseobject.py

```diff
@@ -98,6 +99,21 @@ def update_state(self, new_value, error_message=None):
             self.add_event('state changed',
                            '%s -> %s' % (orig.value, new_value))
 
+    @property
+    def error_msg(self):
+        db_data = self._db_get_attribute('error_msg')
+        if not db_data:
+            return None
+        return db_data
+
+    @error_msg.setter
+    def error_msg(self, msg):
+        if self.state.state != 'error':
+            raise exceptions.InvalidStateException(
+                'Object not in error state (state=%s, object=%s)' % (
+                    self.state, self.object_type))
+        self._db_set_attribute('error_msg', msg)
```

Comment:
I'm not sure how I feel about saving the characters in the database schema, I feel a bit like we'd be better off calling this "error_message", or just "error". It is a bit nit picky though.

Reply:
It felt wasteful to store a dict, but happy to change it to be consistent with other attributes.